Neos.FluidAdaptor/Tests/Unit/ViewHelpers/Format/HtmlentitiesDecodeViewHelperTest.php

```diff
@@ -138,12 +138,12 @@ public function renderDecodesObjectsToStrings()
 
     /**
      * @test
+     * @expectedException \InvalidArgumentException
```

Comment:
Now that's a (breaking?) change in behaviour, isn't it? At any rate the test method name no longer matches the tested behaviour now.

Reply:
Basically because now the validation is moved to standalone Fluid.

Reply:
Mhh... ok, so this will now error out where it previously just silently passed the argument back. Will this have any effect on real world use? i.e. would this have thrown an error later anyway?